src/functions/keycloak/remove-user-attribute/index.ts

```diff
@@ -0,0 +1,16 @@
+import { handlerPath } from "@libs/handler-resolver";
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      httpApi: {
+        method: "delete",
+        path: "/users/{id}/attributes/{attributeName?}",
```

Comment:
Also remember we typically design the new lambda endpoints in the home-lambdas-api-spec first. I haven't seen the spec updates yet so please make sure they get done.
In the spec you will describe the 2 endpoints which should match the path in the handler index files- so in this file for example. Adding in any relevant schemas (I guess none in this case as it doesn't return anything) and the path parameters and their types e.g. the type of attribute.
Also in the spec we should also describe that the only valid attribute name to be updated or removed is the severaId